charts/sysdig/templates/configmap.yaml

```diff
@@ -46,6 +46,8 @@ data:
 {{- if $sslVerifyCertificate }}
     ssl_verify_certificate: {{ $sslVerifyCertificate }}
 {{- end }}
+    k8s_coldstart:
+      max_parallel_cold_starts: {{ include "sysdig.parallelStarts" .}}
```

Comment:
if max_parallel_cold_starts is specified in values.yaml, this will be overwritten by the template in line 52